neo4j-test/doc-tests.js

```diff
@@ -62,21 +62,20 @@ describe('Tests for Documents', function () {
     expect(await getNumEdges()).equal(0);
     await addDocumentToNeo4j(myDoc);
 
-    // todo - refs #1138
-    // expect(await getNumNodes()).equal(2);
-    // expect(await getGeneName('ncbigene:5597')).equal('MAPK6');
-    // expect(await getGeneName('ncbigene:207')).equal('AKT1');
+    expect(await getNumNodes()).equal(2);
+    expect(await getGeneName('ncbigene:5597')).equal('MAPK6');
+    expect(await getGeneName('ncbigene:207')).equal('AKT1');
 
-    // expect(await getNumEdges()).equal(1);
-    // let edge = await getEdge('01ef22cc-2a8e-46d4-9060-6bf1c273869b');
-    // expect(edge.type).equal('INTERACTION');
-    // expect(edge.properties.type).equal('phosphorylation');
-    // expect(edge.properties.sourceId).equal('ncbigene:5597');
-    // expect(edge.properties.targetId).equal('ncbigene:207');
-    // expect(edge.properties.xref).equal(myDoc.id);
-    // expect(edge.properties.doi).equal('10.1126/sciadv.abi6439');
-    // expect(edge.properties.pmid).equal('34767444');
-    // expect(edge.properties.articleTitle).equal('MAPK6-AKT signaling promotes tumor growth and resistance to mTOR kinase blockade.');
+    expect(await getNumEdges()).equal(1);
```

Comment:
Might want to consider in the future comparing to the document instance rather than hardcoding, just thinking if wen end up testing a bunch of different doc cases (e.g. complexes). 

e.g. 
```js
...
   expect(edge.properties.xref).equal(myDoc.id());
    expect(edge.properties.doi).equal(myDoc.citation().doi);
    expect(edge.properties.pmid).equal(myDoc.citation().pmid);
    expect(edge.properties.articleTitle).equal(myDoc.citation().title);
...
```

Reply:
Hmm, that's a good point. 
What about when we have documents to test that aren't binary? Ex. 3 nodes and 2 edges. Should I make different tests for that, or should I have some kind of loop

```
for (let n = 0; n < numNodes; n++) {
  // test this node
}
```

Reply:
@jvwong could you take another look at my test now? I've soft coded all the parts and utilized loops so that the test can handle an arbitrary number of nodes/edges. Let me know what you think!

I was taught in school that tests should be hard coded and I guess that's good for the assignments they gave us, but not for this scenario

Reply:
Works for me. Whatever is easiest to read.